lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -2717,10 +2810,9 @@ TfToken _RequiresColorManagement(
     const HdMaterialNetwork2& inNet)
 #endif
 {
-    const mx::NodeDefPtr nodeDef = _GetMaterialXData()._mtlxLibrary->getNodeDef(node.nodeTypeId);
```

Comment:
We were not using that nodedef, and it was pointing to a missing swizzle in two unit tests, make it business as usual here as well to not have all definitions.

Please note that this applies only when looking at id straight from HdMaterialNetwork data prior to sending them to be converted to a MaterialX document. The returned MaterialX document will have been updated to 1.39 and should not contain missing nodes anymore.